ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/lldp.j2

```diff
@@ -0,0 +1,42 @@
+{% if lldp is arista.avd.defined %}
+
+# LLDP
+
+## LLDP Summary
+
+### LLDP Global Settings
+
+| State | Management Address | Management VRF | Timer | Hold-Time | Re-initialization Timer | Drop Received Tagged Packets |
+| ----- | ------------------ | -------------- | ----- | --------- | ----------------------- | ---------------------------- |
+{%     set row_state = 'Enabled' %}
+{%     if lldp.run is arista.avd.defined(false) %}
+{%         set row_state = 'Disabled' %}
+{%     endif %}
+{%     set row_receive_packet_tagged_drop = 'Disabled' %}
+{%     if lldp.receive_packet_tagged_drop is arista.avd.defined(true) %}
+{%         set row_receive_packet_tagged_drop = 'Enabled' %}
+{%     endif %}
+| {{ row_state }} | {{ lldp.management_address | arista.avd.default('-') }} | {{ lldp.vrf | arista.avd.default('-') }} | {{ lldp.timer | arista.avd.default('30') }} | {{ lldp.holdtime | arista.avd.default('120') }} | {{ lldp.timer_reinitialization | arista.avd.default('2') }} | {{ row_receive_packet_tagged_drop }} |
+
+{%     if ethernet_interfaces is arista.avd.defined %}
+### LLDP Interface Configuration
+
+{%         if lldp.run is arista.avd.defined(false) %}
+LLDP is **disabled** globally. Local interface configs will not apply.
+
+{%         endif %}
+| Interface | Transmit | Receive |
+| --------- | -------- | ------- |
+{%         for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
+{%             set lldp_transmit = ethernet_interfaces[ethernet_interface].lldp.transmit | arista.avd.default("True") %}
```

Comment:
avoid the default here. We should only output interfaces in this table, if lldp has been set specifically on that interface (print both if set to true or set to false). Otherwise you will produce a very long list for all devices.

Reply:
True, will only show the interfaces that have explicit LLDP config.